plugins/flatpak/gs-plugin-flatpak.c

```diff
@@ -533,6 +630,20 @@ gs_plugin_download (GsPlugin *plugin, GsAppList *list,
 			}
 		}
 
+		/* Is there enough disk space to download? */
+		if (!gs_flatpak_has_space_to_update (flatpak, list_tmp, is_auto_update)) {
+			g_debug ("Skipping %s: not enough space on disk",
+			 	 (is_auto_update ? "automatic update" : "update"));
+			if (is_auto_update) {
+				/* Do not return an error because we don't want to see a banner
+				 * showing up out of the blue for possibly multiple update failures */
+				continue;
```

Comment:
The comment above needs a bit readjusting as we are continuing to the next flatpak group and see if they are auto-updatable or not.